src/coreclr/vm/managedmdimport.hpp

```diff
@@ -24,13 +24,6 @@ typedef struct
     void* m_array;
 } ConstArray;
 
-typedef struct
-{
-    I4Array * largeResult;
-    int length;
-    int smallResult[16];
-} MetadataEnumResult;
-
 #define MDDecl0(RET, NAME) static FCDECL1(RET, NAME, IMDInternalImport* pScope)
```

Comment:
Nit: There macros do not provide much value - just insert one argument into the list. I think it would look better without them.